synapse/tools/autodoc.py

```diff
@@ -177,6 +180,12 @@ def processTypes(rst, dochelp, types):
         rst.addLines(doc,
                      f'The ``{name}`` type is derived from the base type: ``{ttyp}``.')
 
+        ifaces = info.pop('interfaces', None)
+        if ifaces:
+            rst.addLines('', f'The ``{name}`` type implements the interfaces:', '')
```

Comment:
```suggestion
            rst.addLines('', f'This type implements the following interfaces:', '')
```

We should revisit the tense here, since it differs from the next section:

```
The meta:aggregate:type:taxonomy type implements the interfaces:

meta:taxonomy

The type meta:aggregate:type:taxonomy has the following options set:

...
```

Reply:
Updated these to both be "this type"